Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to a well-known Ethereum testnet
+   a) support restarting of nodes
+3. Improvements to Management contract
+   a) Support withdrawals
+4. Mempool improvements
+   a) Remove failed transactions
+
+
+## V3
+1. Support a limited number of user operated nodes
+   a) Catching up robustness
+   b) Script robustness
+   c) Instructions
+   d) Try outside of Azure?
```

Comment:
Would v3 have others operate nodes only in Azure? If so, is it a template they just launch?

Reply:
That's something we need to decide. I've listed it here as something to keep in mind.